server.py

```diff
@@ -0,0 +1,20 @@
+import json
+file1 = open("info_about_lunch.txt","r")
```

Comment:
Please use a better name than file1

Reply:
You opened a file but didn't close it. Its leaking memory. Read about memory leakage.
You can also solve it by using a context manager. Read about context managers in Python.

Reply:
I have changed the name of file1
I have also made the file manager class for closing the files.